__tests__/app.test.js

```diff
@@ -215,6 +214,85 @@ describe("GET/api/articles/:article_id/comments", () => {
 	});
 });
 
+describe("POST /api/articles/:article_id/comments", () => {
+	it("POST 201 status code, inserts new comment and responds with posted comment", () => {
+		const newComment = {
+			username: "butter_bridge",
+			body: "This is a new comment",
+		};
+		return request(app)
+			.post("/api/articles/6/comments")
+			.send(newComment)
+			.expect(201)
+			.then(({ body }) => {
+				const comment = body.new_comment[0];
+				expect(comment.comment_id).toBe(19);
+				expect(comment.votes).toBe(0);
+				expect(typeof comment.created_at).toBe("string");
+				expect(comment.author).toEqual(newComment.username);
+				expect(comment.body).toEqual(newComment.body);
+				expect(comment.article_id).toBe(6);
+			});
```

Comment:
Missing one 201 test here: comment successfully created when the request body has unnecessary properties EG: const newComment = {
			username: "butter_bridge",
			body: "This is a new comment",
                         unnecessaryProperty: true
		};
In this case the post should be successful with the unnecessary property ignored